castle/cms/skins/castle/login_form.cpt

```diff
@@ -66,7 +75,7 @@
                 New user?
                 </strong>
 
-                <p i18n:translate="description_no_account">
+                <p tal:condition="bad_domain" i18n:translate="description_no_account">
```

Comment:
does this tal:condition belong?

Reply:
Oops I must have put that in the wrong spot 😩